backend/main/src/Group/UseCases/CreateContainerUseCase/CreateContainerUseCase.ts

```diff
@@ -75,7 +75,15 @@ export class CreateContainerUseCase
 
 		const container = containerOrError.value;
 
+		const newGroupOrError = group.addContainerId(containerId);
+		if (!newGroupOrError.ok) {
+			return Err(newGroupOrError.error);
+		}
+		const newGroup = newGroupOrError.value;
+
 		await this.containerRepository.create(container);
+		// add containerId to the Group
+		await this.groupRepository.create(newGroup);
```

Comment:
Do we need to process this line with blocking?

Reply:
Thank you for your comments! Both comments totally make sense. so I've fixed them.